packages/Sortable/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@paprika/sortable",
+  "version": "0.1.0",
+  "description": "Sortable component",
+  "author": "@paprika",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/acl-services/paprika/tree/master/packages/sortable"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@babel/runtime-corejs2": "^7.3.1",
+    "@paprika/icon": "^0.1.0",
+    "@paprika/stylers": "^0.1.0",
+    "@paprika/tokens": "^0.1.0",
+    "classnames": "^2.2.6",
```

Comment:
why we need className?

Reply:
seem like this package should be add to TextArea and input but not in Sortable?

Reply:
Yeah... whoops.